File: Toggly.FeatureManagement.Flutter/toggly/lib/src/models/toggly_init_response.dart
/// Possible feature flags values loading responses.
enum TogglyLoadFeatureFlagsResponse {
  /// Fetched from the Toggly.io Client API.
  fetched,

  /// Loaded from the cache.
  cached,

  /// Used the feature flag defaults provided during Toggly.init.
  defaults,

  /// Something, somewhere, went wrong.
  error,
}

/// Toggly initialization response model.
class TogglyInitResponse {
  final TogglyLoadFeatureFlagsResponse status;

  TogglyInitResponse({
    required this.status,
  });
}

File: Toggly.FeatureManagement.Javascript/feature_flags_toggly/lib/models/toggly-init-response.ts
export enum TogglyLoadFeatureFlagsResponse {
  fetched,
  cached,
  defaults,
  error,
}

export class TogglyInitResponse {
  status: TogglyLoadFeatureFlagsResponse;

  constructor(status: TogglyLoadFeatureFlagsResponse) {
    this.status = status;
  }
}
